### ISSUE TITLE
Integrate with OSS-Fuzz

### ISSUE THREAD

**Issue by darthShadow** [rclone/rclone#6374]
#### The associated forum post URL from `https://forum.rclone.org`

N/A

#### What is your current rclone version (output from `rclone version`)?

N/A

#### What problem are you are trying to solve?

Integrate with OSS-Fuzz

#### How do you think rclone should be changed to solve that?

https://google.github.io/oss-fuzz/getting-started/accepting-new-projects/

#### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

**Comment by ncw** [comment]
Interesting idea... We'd have to build out fuzz targets as I understand.

Which bits of rclone do you think should be targeted? The servers seem an obvious choice.

**Comment by darthShadow** [comment]
Yeah, the various `serve` commands along with the `rc`  & `rcd` commands should be a good start.

Perhaps the config update commands too?
